sunspot/lib/sunspot/util.rb

```diff
@@ -289,20 +289,40 @@ def sub(*args, &block)
         __proxy_method__(:sub, *args, &block)
       end
 
-      def method_missing(method, *args, &block)
-        __proxy_method__(method, *args, &block)
-      end
+      if RUBY_VERSION >= '3.0'
```

Comment:
Is there something that isn't backwards compatible with Ruby 2.x in here? Could we just use the new kwargs code for all versions?

Reply:
You are right, this code should work with all 2.x versions. I checked it with version 2.7 and it works.